Source/WebCore/Modules/compression/DecompressionStreamDecoder.h

```diff
@@ -73,6 +73,10 @@ class DecompressionStreamDecoder : public RefCounted<DecompressionStreamDecoder>
         : m_format(static_cast<Formats::CompressionFormat>(format))
     {
         std::memset(&m_zstream, 0, sizeof(m_zstream));
+
+#if PLATFORM(COCOA)
+        std::memset(&m_stream, 0, sizeof(m_stream));
```

Comment:
It seems strange that the decoder zero initializes its members but the encoder doesn't.